utils/webhook/github.handler.ts

```diff
@@ -91,10 +108,14 @@ export async function githubWebhookHandler(
         GitHubRepo: { repoName }
     } = sync;
 
-    const linearKey = process.env.LINEAR_API_KEY
+    let linearKey = process.env.LINEAR_API_KEY
         ? process.env.LINEAR_API_KEY
         : decrypt(linearApiKey, linearApiKeyIV);
 
+    if (anonymousUser) {
+        linearKey = process.env.LINEAR_APPLICATION_ADMIN_KEY;

```

Comment:
I wonder if scoping the admin key this broadly could introduce vulnerabilities down the road.

for clarity, should we add a `createAnonLinearComment()` to replace `createLinearComment()`?

Reply:
Mmmm not 100% sure since the only difference is the actual linearKey when initializing the linear SDK everything is exactly the same. What do you mean by vulnerabilities?